packages/contracts-bedrock/justfile

```diff
@@ -31,7 +31,6 @@ build-go-ffi:
 # Removes everything inside of .testdata (except the .gitkeep file).
```

Comment:
Stale comment?

Reply:
fixed in: https://github.com/ethereum-optimism/optimism/pull/12092/commits/17b5bf2626e943faef5fd0426fe815b98b6b5deb